src/coaster/auth.py

```diff
@@ -269,13 +269,13 @@ def init_app(app: Flask) -> None:
 _CurrentAuthType_co = TypeVar('_CurrentAuthType_co', bound=CurrentAuth, covariant=True)
 
 
-class GetCurrentAuth(Generic[_CurrentAuthType_co]):
+class GetCurrentAuth:
```

Comment:
Making this Generic breaks the `proxy` classmethod's return type in Pyright (but not Mypy). Since we only use the proxy, I've removed the Generic here.